README.md

```diff
@@ -47,6 +50,13 @@ PORT=8002 docker-compose up -d --build
 Once completed, you will notice that you have a docker container running on the port you selected, which can process requests just like any django server.
 
 
+## Run tests on Docker
+Once the project has been built with docker using the previous command above. One can launch the unit tests using the following instruction:
+
+```bash
+PORT=8002 docker-compose exec web pyttest .
+```
+
```

Comment:
You think we should add this on top of the section in Contributing? (cf. https://github.com/pyronear/pyronear-api/blob/master/CONTRIBUTING.md#running-ci-verifications-locally)

Reply:
I added this to  the README before your refacto. I removed it in a recent commit.